api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseEncounterTranslator.java

```diff
@@ -37,5 +42,28 @@ protected Coding mapLocationToClass(Location location) {
 		}
 		return coding;
 	}
-	
+
+	protected List<CodeableConcept> mapEncounterTypeClass(EncounterType openmrsEncounterType) {
```

Comment:
We should probably move this to the `EncounterTranslatorImpl`. The primary purpose of the `BaseEncounterTranslator` is to handle the shared functionality between encounters represented as FHIR encounters and visits represented as FHIR encounters (which isn't to say that we shouldn't also have a similar mapping between visit type and the `Encounter.type` field.